src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,28 @@ public override SwitchParameter UseSSL
             }
         }
 
+        /// <summary>
+        /// Gets or sets strict mode.
+        /// </summary>
+        [Experimental(ExperimentalFeature.PSStrictModeAssignment, ExperimentAction.Show)]
+        [Parameter(ParameterSetName = InvokeCommandCommand.InProcParameterSet)]
+        [ArgumentToPSVersionTransformation]
+        [ValidateVersion]
```

Comment:
@Thomas-Yu  Tests are failing because `version == 0.0` is an invalid PSVersion.  So I think we will need to remove this `[ValidateVersion]` attribute and validate the StrictMode parameter in code.  Or, alternatively you could create a new version validation attribute, such as `ValidateVersionOffAttribute` that allows a version value of '0.0'.

I think the logic will be something like:
```csharp
private static readonly Version s_OffVersion = new Version(0, 0);
if (version == null || (!PSVersionInfo.IsValidPSVersion && version != s_OffVersion)) { throw ... }
```
```

Reply:
> Would allowing -StrictMode off essentially also allow -StrictMode 0.0?

Maybe we want to keep consistent with `Set-StrictMode` and also disallow `0.0` here? Also, given that we have `-StrictMode off`, we don't need a user to use `0.0`, which is obscure IMO.

Reply:
We need to change it in any case because the conversion attribute sets it to 0.0 and that is triggering the error.

Reply:
Ah I see. In that case, I would vote for 2 parameters `-ScriptMode` and `-ScriptModeOff`, as @PaulHigin suggested before. So no change is needed for the attributes.

Reply:
The committee already said they preferred the `Off` string value conversion, so I feel we should implement it.  I feel the best solution is to perform validation in code, or create a new `ValidateVersionOff` attribute that also allows 0.0 value.

Reply:
Got it.
@Thomas-Yu Then please keep `ArgumentToPSVersionTransformationAttribute` and `ValidateVersionAttribute` unchanged, and create new private attribute classes for this parameter.